### DISCUSSION TITLE
Our own CONTRIBUNTING !

### DISCUSSION THREAD

**Discussion by aneojgurhem** [aneoconsulting/ArmoniK#865] [Ideas]
In this post, I want to share (and remember) the good contributing examples I found in the Open Source community. This can help to build our own. Do you have any good examples ? What do you think of the following one ?

- https://github.com/dotnet/efcore/blob/main/.github/CONTRIBUTING.md
- https://github.com/nhibernate/nhibernate-core/blob/master/CONTRIBUTING.md

**Comment by Barbapapazes** [comment]
Here some contributing.md

https://github.com/elk-zone/elk/blob/main/CONTRIBUTING.md

https://github.com/antfu/contribute
